sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -177,7 +182,7 @@ export class Recorder {
     this.stateManager.state = "stopped";
     if (this.recordingId !== undefined) {
       const stopUri = `${this.url}${isPlaybackMode() ? paths.playback : paths.record}${paths.stop}`;
-      const req = this._createRecordingRequest(stopUri);
+      const req = createRecordingRequest(stopUri, this.sessionFile, this.recordingId);
```

Comment:
Talked to @scbedd recently on this.
He said the sessionFile is not needed for the stop request.

Reply:
It's optional already, just assumed that we needed it passed in since it was being passed in implicitly before. I can just pass through `undefined` instead.